src/sanitize-argument.ts

```diff
@@ -180,9 +181,10 @@ export function sanitize_speed(_speed: any) {
   }
 }
 
-export async function check_mwUrl(mwUrl: string) {
-  await Downloader.get(mwUrl).catch((err) => {
-    throw new Error(`mwUrl [${mwUrl}] is not valid:\n${err}`)
+export async function check_mwApiReachability(mwUrl: string, mwActionApiPath: string) {
+  const apiFullUrl = new URL(mwActionApiPath || MediaWiki.actionApiPath, mwUrl).toString()
+  await Downloader.get(apiFullUrl).catch((err) => {
+    throw new Error(`Mediawiki API URL [${apiFullUrl}] is not reachable:\n${err}`)
```

Comment:
Not sure if this would be too much at this point, but we could hit `?action=query&format=json&formatversion=2` to ensure there is actually a MediaWiki Action API endpoint here.